crt_portal/static/js/form_letter.js

```diff
@@ -1,4 +1,39 @@
 (function(root) {
+  const DEPT_ADDRESS = {
+    deptOfEd:
+      '<p id="form-letterhead--dept-addressee">U.S. Department of Education<br>Office for Civil Rights<br>Lyndon Baines Johnson Department of Education Bldg.<br>400 Maryland Avenue, SW<br>Washington, DC 20202-1100<br></p>',
+    DOT:
+      '<p id="form-letterhead--dept-addressee">Director of Civil Rights Advocacy<br>Aviation Consumer Protection Division<br>Department of Transportation<br>1200 New Jersey Avenue, S.E., C-75<br>W96-432<br>Washington, D.C.  20590<br></p>',
+    HHS:
+      '<p id="form-letterhead--dept-addressee">Office for Civil Rights<br>Department of Health and Human Services<br>200 Independence Avenue, SW, Room 515F<br>Humphrey Building<br>Washington, D.C.  20201<br></p>',
+    EEOC:
+      '<p id="form-letterhead--dept-addressee">Field Management Programs<br>U.S. Equal Employment Opportunity Commission<br>131 M Street, N.E.<br>Washington, DC  20507<br></p>'
+  };
+
+  const addReferralAddress = option => {
+    let addressee = document.getElementById('form-letterhead--addressee');
+    let deptAddressee = document.getElementById('form-letterhead--dept-addressee');
+    if (deptAddressee) {
+      deptAddressee.remove(deptAddressee);
```

Comment:
you don't need to pass it a reference to itself! `.remove()` is automatically called on itself. e.g.

```js
deptAddressee.remove()
```